pkg/rancher-components/src/components/Form/Checkbox/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Checkbox } from './Checkbox.vue';
```

Comment:
I am not sure to understand the scope of creating an empty TS file with another export.

Reply:
Doing something like this provides us with two benefits:

1. This provides us with a way to perform named exports without disrupting the requirement that Vue components be the default export. In essence, this allows us to `import { Checkbox, Cart, LabeledInput, etc.. } from '@rancher/components'` after we publish
2. It's an organizational technique to manage component exports close to the source. If we focus on Radio Buttons and form as an example, we can see that `RadioButton` & `RadioGroup` are exported via `pkg/rancher-components/src/components/Form/Radio/index.ts`. This export passed all the way up to `pkg/rancher-components/src/index.js`

An alternative might be to maintain a long list of exports at the project root. I didn't perceive that to be very nice to maintain, especially with the number of potential components that we could be publishing. This technique was influenced by other frameworks like Vuetify.

Reply:
Ok this makes totally sense, to have some barrel imports. I was trying to replace the path to see if on fact was that the scope but I had issues about it.
Are you sure is going to work later as library? Also why is not working already with the existing export?

Reply:
This works using a library. I use the library is my primary method of testing, it is the goal of this PR after all 😁

Anything we're doing here with these aliases and reaching directly into `pkg/rancher-components` for imports is just an intermediary step until we can get to a point where we can publish our first version of the component library